ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.4.yaml

```diff
@@ -52,37 +52,30 @@ tests:
 - as: cargo-test
   steps:
     pre:
-    - as: ensure-vendoring
-      commands: dist/vendor.sh && git diff --quiet
-      from: src
```

Comment:
Why are we dropping this?  Not that I care all that much about the 4.4 branch anymore, just curious.

Reply:
Right, lets leave it there